kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -334,6 +337,25 @@
       updateContentState(contentState, forceSave = true) {
         this.updateContentNodeState({ contentState, forceSave });
       },
+      navigateToRegularContext(message) {
+        let id = message.nodeId;
+        return ContentNodeResource.fetchModel({ id })
+          .then(contentNode => {
+            let routeBase, path;
+            if (contentNode && contentNode.kind !== 'topic') {
```

Comment:
Is the `contentNode.isLeaf` field available from this endpoint? If so, this should also work:

```suggestion
            if (contentNode && contentNode.isLeaf) {
```

Reply:
Yes, should be.